packages/kotti-ui/source/kotti-avatar/KtAvatar.vue

```diff
@@ -47,7 +47,7 @@ export default defineComponent({
 			computed(() => ({
 				appendTo: () => document.body,
 				arrow: roundArrow,
-				content: contentRef.value as NonNullable<typeof contentRef.value>,
+				content: contentRef.value ?? undefined,
```

Comment:
useTippy has an `onMounted` hook inside it that only binds the tippy instance when the component mounts

so IN THEORY, tippy isn't bound unless the component mounts which means the content shouldn't be null

However, since tippy seems to handle undefined content just fine, we unified all the usages to fallback to undefined if contentRef.value is still null